xml/Microsoft.Build.Tasks/CreateProperty.xml

```diff
@@ -161,8 +161,8 @@
                         Output TaskParameter="Value" PropertyName="MyTargetsToBuild"
                     /CreateProperty
             
-            We need to respect the semicolon that they put in the value, and need to treat
-            this exactly as if they had done:
+            We need to respect the semicolon that he put in the value, and need to treat
+            this exactly as if he had done:
```

Comment:
```suggestion
            this exactly as if they had done:
```

Reply:
.NET docs did a pass for inclusive pronouns, so adding that change to the original source comments as well
@rainersigwald